pkg/operator/target_config_reconciler.go

```diff
@@ -1046,6 +1282,8 @@ func (c *TargetConfigReconciler) manageConfigMap(descheduler *deschedulerv1.Kube
 		case deschedulerv1.CompactAndScale:
 			profile, err = compactAndScaleProfile(descheduler.Spec.ProfileCustomizations, includedNamespaces, excludedNamespaces, ignorePVCPods, evictLocalStoragePods)
 		case deschedulerv1.RelieveAndMigrate:
+			kubeVirtShedulable := "kubevirt.io/schedulable=true"
+			policy.NodeSelector = &kubeVirtShedulable
```

Comment:
Can you also mention this in the README? So users are aware the node selector is always set when the profile is used.

Reply:
Done